src/backend/u64/mod.rs

```diff
@@ -1,4 +1,3 @@
-#[cfg(not(any(feature = "u64_backend")))]
```

Comment:
Why this is being removed? This is the exact reason why this PR is not compiling. 
Here we force the compiler to use the `u64_feature` if none is selected. 

Please, this line has to be added again, there is absolutely no reason to remove this.

Reply:
The issue stated above have been addressed and fixed.

Reply:
Thanks!